packages/codegen/src/config.ts

```diff
@@ -140,6 +140,9 @@ export const config = z
     allowImportingTsExtensions: z.optional(z.boolean()),
     // The version of Joist that generated this config.
     version: z.string().default("0.0.0"),
+
+    /** A list of postgres schemas to use for finding tables */
+    includeSchema: z.optional(z.array(z.string())),
```

Comment:
@brudil wdyt about calling this `schemas`?

Reply:
No quarms here - was torn on a hundred possibilities.

Reply:
Updated in [a001e36](https://github.com/joist-orm/joist-orm/pull/1406/commits/a001e36b6dc67241ac04b4732308136946d62d6c)